src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Skellige/Silver/KnutTheCallous.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Alsein.Extensions;
+using System;
+
+namespace Cynthia.Card
+{
+    [CardEffectId("70080")]//冷酷无情的克努特 KnuttheCallous
+    public class KnuttheCallous : CardEffect
+    {//Deploy: discard a card from your hand, then generate a bronze unit from your starting deck, strengthen it by 1 and add it to your hand
```

Comment:
Actually, I am a bit afraid of this card being too strange, like the ArtoriusVigo in Nifgaar

Reply:
It's meant to be Skellige's Artorius Vigo yes, several of us on Discord felt that the card would fit much better in Skellige, because of discard synergy and because it doesn't have as abusable bronzes as nilfgaard. I'm also in favor of keeping card effects simple, but I figured it would not surprise anyone since it is already in the game through Vigo. The card was designed with Discard and Queensguards in mind, the discussions on Discord agreed that it would probably be used in veterans, but not likely in greatswords or axemen. What do you think?

Reply:
OK, I guess we can try that. But after updating, if this card is abused too widely, we should remove it.